src/utils/addData.js

```diff
@@ -0,0 +1,55 @@
+const inquirer = require("inquirer");
+const {
+  getDepartmentQuestion,
+  getRoleQuestion,
+  getEmployeeQuestions,
+} = require("../questions");
+
+const newDepartment = async (executeQuery) => {
+  const { department } = await inquirer.prompt(getDepartmentQuestion());
+
+  await executeQuery(
+    `INSERT INTO department (dep_name) VALUES ("${department}")`
+  );
+
+  const message = `${department} Department Added`;
```

Comment:
no need to declare a variable just to return it. We can simply return the template string.